hack/util/checkhash_helm.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The cert-manager Authors.
```

Comment:
If this is new, 2023?

Reply:
good point I should update that, thanks!